inst/templates/Makefile

```diff
@@ -0,0 +1,29 @@
+R = R --vanilla CMD
+
+
+all: build check install clean
+
+build:
+	$(R) build .
+
+check: build
+	$(R) check --no-manual *.tar.gz
+
+define devtools_check
+	Rscript -e " \
+		tryCatch( \
+			invisible(packageVersion('devtools')), \
+			error = function(e) { \
+				install.packages('devtools', repos = 'https://cloud.r-project.org') \
+			} \
+		)"
+endef
+install:
+	@$(devtools_check); \
+	Rscript -e 'devtools::install()'
```

Comment:
Rather than using devtools maybe use remotes, break this into a separate rule and install from the built tarball.

```make
install_deps:
	Rscript -e 'if (!requireNamespace("remotes") install.packages("remotes")' -e 'remotes::install_deps(dependencies = TRUE)'

install: install_deps build
	R CMD INSTALL $(PKGNAME)_$(PKGVERS).tar.gz
```